src/com/google/javascript/jscomp/CodingConventions.java

```diff
@@ -304,6 +306,17 @@ public ObjectLiteralCast getObjectLiteralCast(Node callNode) {
 
     private static final long serialVersionUID = 1L;
 
+    private static final Set<String> propertyTestFunctions;
```

Comment:
Can you switch this to use ImmutableSet.of, like this:
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/ClosureCodingConvention.java#L345